.bazelrc

```diff
@@ -1,3 +1,3 @@
 # We can't create a bzl_library for rules-swift because of its visibility,
 # so circumvent by not using the sandbox
-common --strategy=Stardoc=standalone
+build --strategy=Stardoc=standalone
```

Comment:
Why does this setting need to change?

Reply:
`--strategy` is apparently not valid for `query`

Reply:
thx for fixing this! I just ran into it